test/unittest/tests/utils.cpp

```diff
@@ -14,6 +14,12 @@
 
 namespace utils {
 
+/*
+ * throwIfNonZero --
+ *     Test result. Throw if it is non-zero.
+ *
+ * @param result The value to test.
+ */
 void
 throwIfNonZero(int result)
```

Comment:
We should be using snake case rather than camel case in C++. I know that previous C++ files may have things in camel case, but I think we should try to re-style the functions that we change or use at least.

Reply:
Hi @mm-ng , I completely agree that we should rename these helper functions to the current coding standard. To avoid making this PR bigger than necessary though, I suggest we create a new ticket to rename the functions in `utils.cpp` which we can then do soon. Monica, could I ask you to create that ticket? And I think it sense to do the renaming step-by-step rather than all in one go, but I'm happy to discuss that. Thanks, Jeremy

Reply:
Hi @mm-ng , thank you, and it turns out that @jiechenbo has already created one (thanks @jiechenbo )! See https://jira.mongodb.org/browse/WT-13405